src/Controls/samples/Controls.Sample/XamlApp.xaml.cs

```diff
@@ -48,7 +51,8 @@ async void LoadAsset()
 		// Must not use MainPage for multi-window
 		protected override Window CreateWindow(IActivationState? activationState)
 		{
-			var window = new MauiWindow(Services.GetRequiredService<Page>())
+			var services = activationState!.Context.Services;
+			var window = new MauiWindow(services.GetRequiredService<Page>())
```

Comment:
This has to be changed because the `IServiceProvider` in the app's ctor is actually the root povider and does not have a window dispatcher.

The page uses a VM that uses the dispatcher. However, the services coming in here via the activation state _is_ a window provider and _does_ have the dispatcher.